pynest/nest/tests/test_aeif_lsodar.py

```diff
@@ -281,11 +281,11 @@ def test_closeness_nest_lsodar(self):
         self.assert_pass_tolerance(rel_diff, di_tolerances_lsodar)
 
     @unittest.skipIf(not HAVE_GSL, 'GSL is not available')
-    def test_iaf_behaviour(self):
-        '''
-        The models should behave as iaf_cond_* if a == 0., b == 0. and
-        Delta_T == 0.
-        '''
+    def testIafBehaviour(self):
+
+        # The models should behave as iaf_cond_* if a == 0., b == 0. and
+        # Delta_T == 0.
```

Comment:
I think I misunderstood. So even if nosetest only outputs a single line that still breaks our output parsing? I had though/hoped that our parsing would work as long as nose did no insert any line breaks. We should discuss how to proceed in the next VC.